### ISSUE TITLE
Hold down on video player to play video in 2x speed.

### ISSUE THREAD

**Issue by Jordan-L77** [futo-org/grayjay-android#1369]
### Use Cases

When holding down on video player the video plays in 2x speed until you stop holding down. I will attach a video as an example.

https://github.com/user-attachments/assets/b31212d7-04bd-493f-9590-2ba362d00633

**Comment by MKBontwikkeling** [comment]
I'd really love this too. Hard to implement?

**Comment by obakha** [comment]
Hi man, this would be a great if you implented it in the video player!

**Comment by Timoite** [comment]
Will it ever be implemented?

**Comment by Zvonimir-FUTO** [comment]
Yes! Will see when we can fit that in but had to focus on other stuff. I also really want that feature in. Time is the only constraint.

**Comment by flerouwu** [comment]
I've noticed this feature appears to have recently been added in the Android app (although I cannot find anything in the changelog about it).

Is there any way to disable this feature? I wasn't able to find anything in settings.

**Comment by Zvonimir-FUTO** [comment]
Added in v327.
